x/wasm/internal/types/keys.go

```diff
@@ -96,3 +97,12 @@ func GetContractCodeHistoryElementPrefix(contractAddr sdk.AccAddress) []byte {
 	copy(r[prefixLen:], contractAddr)
 	return r
 }
+
+// GetPinnedCodeIndexPrefix returns the key prefix for a code id pinned into the wasmvm cache
+func GetPinnedCodeIndexPrefix(codeID uint64) []byte {
+	prefixLen := len(PinnedCodeIndexPrefix)
+	r := make([]byte, prefixLen+sdk.AddrLen)
```

Comment:
Uh, this should be prefixLen + 8, right? (64 bytes)

Reply:
Ups, right. Leftover from previous version 🙈